packages/design-system/src/molecules/tokenIdentification/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Pressable, useBreakpointValue } from 'native-base'
+import { View } from 'react-native'
+import { CircleImg } from '../../atoms'
+import { TypographyV2 } from '../../atoms/typographyV2'
+import { useThemeV2 } from '../../hooks'
+import { container, margin } from '../../styles'
+
+interface TokenIdentificationProps {
+  logo: string
+  name: string
+  symbol: string
+  action: {
+    title: string
+    onPress: () => void
+  }
+}
+export const TokenIdentification: React.FC<TokenIdentificationProps> = ({
```

Comment:
Isn't it a replace of `atoms/tokenChip`? Can we have one component for any token representation?

Reply:
I'd use a more declarative name for this component. It does not identify a token. It renders a token's visualization. Wdyt about `TokenAppearance`?